### PR TITLE
Add missing V10 updates to the docs

### PR DESCRIPTION
**Pull request by bartekn**

### CODE REVIEW

File: services/horizon/internal/docs/reference/resources/effect.md [hunk 1]
```diff
@@ -63,6 +64,12 @@ We can distinguish 5 effect groups:
 | Data Removed | manage_data |
 | Data Updated | manage_data |
 
+### Data effects
```
Comment (robertDurst) [inline services/horizon/internal/docs/reference/resources/effect.md]: Is this supposed to be misc effects? Also, small and nit picky, but above it says there are 5 effects, there are now six :)

Comment (bartekn) [reply services/horizon/internal/docs/reference/resources/effect.md]: Yeah, I wasn't sure about it. I feel like, it doesn't need a whole new category for a single effect. We can always change it later ;). Updated the number.

Comment (robertDurst) [reply services/horizon/internal/docs/reference/resources/effect.md]: Ok makes sense! Also last thing the header here doesn't match.

Comment (bartekn) [reply services/horizon/internal/docs/reference/resources/effect.md]: Fixed.

**Pull request merged**